src/lib/contexts/TalentContext.tsx

```
@@ -0,0 +1,274 @@
+    case "SET_TALENTS":
+      return { ...state, talents: action.payload, filteredTalents: action.payload }
+    case "SET_FILTERED_TALENTS":
+      return { ...state, filteredTalents: action.payload }
+    case "SET_SELECTED_TALENT":
+      return { ...state, selectedTalent: action.payload }
+    case "SET_FILTERS":
+      return { ...state, filters: { ...state.filters, ...action.payload } }
+    case "SET_SEARCH_QUERY":
+      return { ...state, searchQuery: action.payload }
+    case "TOGGLE_SAVE_TALENT":
+      const talentId = action.payload
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)